pkg/logs/config/integration_config.go

```diff
@@ -58,7 +59,8 @@ type LogsConfig struct {
 	Type string
 
 	Port int    // Network
-	Path string // File
+	Path string // File, Journald
+	Unit string // Journald
```

Comment:
@mnshdw had thoughts about how we could avoid having one config object with fields for all integrations. Maybe now is the right time to tackle that?